faust/stores/__init__.py

```diff
@@ -10,6 +10,7 @@
 STORES: FactoryMapping[Type[StoreT]] = FactoryMapping(
     memory="faust.stores.memory:Store",
     rocksdb="faust.stores.rocksdb:Store",
+    aerospike="faust.stores.aerospike:AeroSpikeStore",
```

Comment:
I'm still learning about Aerospike, but does it extend an abstract datastore format that we can use here so that users can _choose_ to use Aerospike as an implementation? (Eg. MariaDB v. MySQL)

However, I don't know if that's realistic when it comes to datastores compared to services.

Reply:
you can plugin your custom adapter using this framework